config.h.cmake

```diff
@@ -0,0 +1,17 @@
+/* config.h.  Generated from config.h.cmake by cmake  */
+
+/* Name of package */
+#define PACKAGE "libofx"
+
+/* Version number of package */
+#define VERSION "@LIBOFX_VERSION_RELEASE_STRING@"
+
+#cmakedefine _WIN32 1
```

Comment:
I still don't understand why _WIN32 is being defined in our build system. If we're on windows, it exists anyway, see https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170 for MSVC and https://blog.kowalczyk.info/article/j/guide-to-predefined-macros-in-c-compilers-gcc-clang-msvc-etc..html for an overview for all systems.

Reply:
Ah yes, forgot to remove it.

Reply:
Fixed.